app/project/[id]/components/project-files.tsx

```diff
@@ -154,13 +204,20 @@ function FilesTable({
                     key={header.id}
                     className="p-2 first:pl-8 last:pr-8"
                     style={{ width: `${header.getSize()}rem` }}
+                    onClick={header.column.getToggleSortingHandler()}
                   >
-                    {header.isPlaceholder
-                      ? null
-                      : Tan.flexRender(
+                    {header.isPlaceholder ? null : (
+                      <div className="flex items-center cursor-pointer">
+                        {Tan.flexRender(
                           header.column.columnDef.header,
                           header.getContext(),
                         )}
+                        {{
+                          asc: <ArrowUp className="w-3.5 h-3.5" />,
+                          desc: <ArrowDown className="w-3.5 h-3.5" />,
+                        }[header.column.getIsSorted() as string] ?? null}
```

Comment:
interesting solution! I prefer these to be in the same place as the header and for the icon and the name to be in a button together, e.g.
`    header: ({ header }) => <Button>Filename {header.column.getIsSorted() ? ....}</Button>,`

Reply:
If I add sorting functionality in Table header from one place, it is reducing the repeated code. I think the approach is good, but if you prefer and there is any reason please tell me shortly, I will add the code in your prefer area.